src/render/vitagxm/SDL_render_vita_gxm.c

```diff
@@ -269,8 +269,11 @@ VITA_GXM_CreateRenderer(SDL_Window *window, Uint32 flags)
     renderer->driverdata = data;
     renderer->window = window;
 
-    if (data->initialized != SDL_FALSE)
-        return 0;
+    if (data->initialized != SDL_FALSE) {
```

Comment:
Is this check even sensible? `data` is allocated by `calloc` above, therefore its `initialized` field always will be 0 at this point, no? Or am I missing something?

Reply:
Yeah, this check is useless

Reply:
Then just remove it.

And please squash your commits into a single one and force-push:
Otherwise the git history will get messy.

Reply:
> And please squash your commits into a single one and force-push: Otherwise the git history will get messy.

I just rebased and squashed your commits and force-pushed.

Reply:
Thanks!